packages/circuit-ui/components/Selector/Selector.tsx

```diff
@@ -68,48 +76,57 @@ export interface SelectorProps extends HTMLProps<HTMLInputElement> {
   tracking?: TrackingProps;
 }
 
-type LabelElProps = Pick<SelectorProps, 'disabled' | 'noMargin'>;
+type LabelElProps = Pick<SelectorProps, 'disabled' | 'noMargin' | 'size'>;
+
+const LABEL_BORDER_RADIUS = '8px';
+
+/**
+ * ::before serves as a host element for the focus outline.
+ * Selector uses box-shadow to draw its border, therefore
+ * in order to draw focus outline with another box-shadow
+ * we need a different element.
```

Comment:
Let's do this with multiple comma-separated box-shadows, it will be cleaner 🙂 I managed to get it to work locally on your branch, I'll show you my thinking on our call today